tests/enum.lisp

```diff
@@ -130,13 +130,30 @@
   3.42
   4.42)
 
+;; Test undeclared values
+(defcenum (enum.undeclared :int :undeclared-values-allowed-p T)
+  (:one 1)
+  (:two 2))
+
+(deftest enum.undeclared
+    (with-foreign-object (enum 'enum.undeclared 3)
+      (setf (mem-aref enum 'enum.undeclared 0) 1)
+      (setf (mem-aref enum 'enum.undeclared 1) 2)
+      (setf (mem-aref enum 'enum.undeclared 2) 3)
+      (values (mem-aref enum 'enum.undeclared 0)
+              (mem-aref enum 'enum.undeclared 1)
+              (mem-aref enum 'enum.undeclared 2)))
+  :one
+  :two
+  3)
+
 ;;;# Bitfield tests
 
 ;;; Regression test: defbitfield was misbehaving when the first value
 ;;; was provided.
 (deftest bitfield.1
     (eval '(defbitfield (bf1 :long)
-             (:foo 0)))
+            (:foo 0)))
```

Comment:
Revert this? Or maybe change the quote to backquote to get proper automatic indentation?